### ISSUE TITLE
Доработка package-loader

### ISSUE THREAD

**Issue by theshadowco** [EvilBeaver/oscript-library#161]
Прошу рассмотреть возможность доработки скрипта, чтобы он мог нормально читать файл при наличии комментариев в xml.
Например

	<package-def>
		
    	<class name="МойКласс" file="src/Core/Классы/МойКласс.os"/>
		<module name="МойМодуль" file="src/Core/Модули/МойМодуль.os"/>
    	
		<module name="МойПриватныйМодуль" file="src/Core/Модули/МойПриватныйМодуль.os"/>
	</package-def>

На данный момент при такой конструкции файла загрузка его не происходит из-за алгоритма чтения
`Пока Чтение.Прочитать() и Чтение.ТипУзла = ТипУзлаXML.НачалоЭлемента Цикл`

**Comment by theshadowco** [comment]
Либо, как вариант, в принципе внести доработку для возможности включения в библиотеку приватных файлов, доступных только в рамках класса

**Comment by artbear** [comment]
ИМХО без комментариев смотрится лучше и проще (добавил атрибут private)
```xml
<package-def>
	<class name="МойКласс" file="src/Core/Классы/МойКласс.os"/>
	<module name="МойМодуль" file="src/Core/Модули/МойМодуль.os"/>

	<module name="МойПриватныйМодуль" file="src/Core/Модули/МойПриватныйМодуль.os"
		private="true"/>
</package-def>
```

А наличие комментариев в тексте реализовать отдельно, не привязываясь к приватности классов/модулей.

**Comment by theshadowco** [comment]
@artbear Да, без них лучше, но обрабатывать корректно их стоит... могу пулреквест сделать

**Comment by artbear** [comment]
От PR хуже не станет, так что лучше сделай.

Правда, по приватным модулям/классам нужно на уровне движка думать.
У нас сейчас нет приватности для классов и модулей, они все доступны в любом скрипте.

@EvilBeaver @dmpas Поправьте меня, если я ошибаюсь.

**Comment by EvilBeaver** [comment]
А мне вот интересно, 1С как комментарии обрабатывает? Старый код в 1С также будет не работать? Просто, возможно, это несовместимость ЧтенияXML с платформой и надо править движок?
